docs/operation-guides/10-backup-kyma.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Back up Kyma
+type: Backups
```

Comment:
I guess you are still using `Type` only temporarily to be able to render the content?

Reply:
Correct :) 
It should be obsolete by now